FlowCrypt/Functionality/Services/Compose Message Service/ComposeMessageAttachment.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  ComposeMessageAttachment.swift
+//  FlowCrypt
+//
+//  Created by Yevhen Kyivskyi on 24.09.2021.
+//  Copyright © 2021 FlowCrypt Limited. All rights reserved.
+//
+
+import UIKit
+import Photos
+
+struct ComposeMessageAttachment {
+    let name: String
+    let size: Int
+    let data: Data
+    let type: String
+}
+
+extension ComposeMessageAttachment {
+    init?(librarySourceMediaInfo: [UIImagePickerController.InfoKey: Any]) {
+        guard let image = librarySourceMediaInfo[.originalImage] as? UIImage,
+              let data = image.pngData(),
+              let asset = librarySourceMediaInfo[.phAsset] as? PHAsset else {
+            return nil
+        }
+        let assetResources = PHAssetResource.assetResources(for: asset)
+
+        guard let fileName = assetResources.first?.originalFilename else {
+            return nil
+        }
+
+        self.name = "\(fileName).pgp"
+        self.data = data
+        self.size = data.count
+        self.type = "image/jpg"
```

Comment:
This is a guess - it's not good to guess. The image may be one of many type of images, not just a jpeg. Real type should be put here to reflect the image. Also above it mentiones `pngData` so there's a mismatch there too.

Reply:
I checked with gmail app, they sending JPG data, so I fixed